app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.java

```diff
@@ -207,8 +208,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     switch (item.getItemId()) {
       case R.id.select_language:
-        if (mViewPager.getCurrentItem() == 1)
+        if (mViewPager.getCurrentItem() == 1) {
+         if(mSectionsPagerAdapter.libraryFragment.libraryAdapter.languages.size()==0)
```

Comment:
Small formatting issue. Please use {} for if statements and also spaces so .size() == 0) {